### ISSUE TITLE
Show log files using a 'dummy' table-less model results in: "undefined method `find' for Log:Class"

### ISSUE THREAD

**Issue by phlegx** [activeadmin/activeadmin#2281]
I wanted to show some log files via the activeadmin interface. Since the log files do not have db tables but are only files I tried to create a table-less model like this:

<pre>
class Log
  include ActiveModel::Validations
  include ActiveModel::Conversion
  extend ActiveModel::Naming
  
attr_accessor :file
end 
</pre>

within app/admin/logs.rb file I then want to present the show view like this:

<pre>
ActiveAdmin.register Log do
  show do
    ...
  end
end
</pre>

but this results in the error:

<pre>
2013-06-18 13:03:12.117 [fatal] NoMethodError - undefined method `find' for Log:Class:
  activeadmin (0.6.0) lib/active_admin/resource_controller/data_access.rb:113:in `find_resource'
  activeadmin (0.6.0) lib/active_admin/resource_controller/data_access.rb:89:in `resource'
  inherited_resources (1.4.0) lib/inherited_resources/actions.rb:13:in `show'
  activeadmin (0.6.0) lib/active_admin/resource_controller/actions.rb:24:in `show'
</pre>

Now it is somewhat clear to me that this will not work since I do not have a table and activeadmin wants to read from the db table. However I'm not so sure about the error and where it does come from. Which 'find' method is this missing one?

So basically my question is: How can I render a show view within activeadmin with a table-less model (ie. via path and parameter:  'admin_log_path(file)')? Is there a 'hack' for my approach, or is this, conventional approach here completely wrong and do I have to do this in a complete different way?

thank you so much
p
